hyppo/ksample/mean_embedding.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from numba import jit
+from scipy.stats import chi2
+from ._utils import _CheckInputs
+from .base import KSampleTestOutput, KSampleTest
+
+
+class MeanEmbeddingTest(KSampleTest):
+    r"""
+        Mean Embedding test statistic and p-value
+
+        The Mean Embedding test is a two-sample test that uses
```

Comment:
make use of the 88 line limit from Sphinx

Reply:
Limited lines to 88 characters